src/components/Image/Image.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Image from 'react-bootstrap/Image';
+import '../../style.css';
+
+export interface ImageWETProps {
+  /** Sets image shape as rounded. */
+  rounded?: boolean;
+  /** Sets image shape as circle. */
+  roundedCircle?: boolean;
+  /** Sets image shape as thumbnail. */
+  thumbnail?: boolean;
+  /** Sets the source of the image */
+  src?: string;
+  /** Sets the alternate text for the image */
+  alt?: string;
+}
+
+const ImageWET = ({
+  rounded = false,
+  roundedCircle = false,
+  thumbnail = false,
+  src = '',
```

Comment:
if you look at the style guide there are a few more options that we should integrate:
https://wet-boew.github.io/wet-boew-styleguide/design/images-en.html

Reply:
I will add these today